hail/src/main/scala/is/hail/annotations/Region.scala

```diff
@@ -45,10 +45,15 @@ final class Region() extends NativeBase() {
   @native def nativeAlign(alignment: Long): Unit
   @native def nativeAlignAllocate(alignment: Long, n: Long): Long
   @native def nativeAllocate(n: Long): Long
+  @native def nativeAddReferenceTo(r2: Region): Unit
+  @native def nativeGetNewRegion(): Unit
   
   final def align(a: Long) = nativeAlign(a)
   final def allocate(a: Long, n: Long): Long = nativeAlignAllocate(a, n)
   final def allocate(n: Long): Long = nativeAllocate(n)
+
+  final def reference(other: Region): Unit = nativeAddReferenceTo(other)
+  final def refreshRegion(): Unit = nativeGetNewRegion()
```

Comment:
it's a bit confusing to have `refreshRegion` in scala and `getNewRegion` in c++. Thoughts on unifying the names?

Reply:
same for reference and addReferenceTo